### ISSUE TITLE
Dropout: a simple way to prevent neural networks from overfitting.

### ISSUE THREAD

**Issue by NorihiroNakaoka** [NorihiroNakaoka/papers#19]
## 0. 論文
[Dropout: a simple way to prevent neural networks from overfitting.](https://www.cs.toronto.edu/%7Ehinton/absps/JMLRdropout.pdf)
Srivastava, Nitish, et al.
